### PR TITLE
Update i18n.js

### PR DESCRIPTION
**Pull request by flaviedesp**
Ajout de la langue francaise

### CODE REVIEW

File: auto_py_to_exe/web/js/i18n.js [hunk 1]
```diff
@@ -221,6 +240,7 @@ const translationMap = {
                 sp: 'Consola de Windows (Ocultar consola)',
                 tr: 'Pencere Tabanlı (Konsolu gizleyin)',
                 th: 'หน้าต่าง (ซ่อนคอนโซล)',
+                fr: 'Basé sur windows (la console n''est pas visible')',
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:243]: Looks like the last quote has not been escaped correctly. Have you run this yourself?

File: auto_py_to_exe/web/js/i18n.js [hunk 2]
```diff
@@ -855,5 +921,9 @@ const supportedLanguages = [
         name: 'Turkish (Türkçe)',
         code: 'tr',
     },
+    {
+        name: 'French (Français)',
+        code: 'fr',
+    },
```
Comment (brentvollebregt) [inline auto_py_to_exe/web/js/i18n.js:927]: Can we have this above "Chinese Traditional" please (alphabetical after English)

**Review by brentvollebregt — CHANGES_REQUESTED** [review]
Just a couple of comments. Can you also add an entry to the translations table in the REAMDE.md file?

File: auto_py_to_exe/web/js/i18n.js (hunk 1 above)
Comment (flaviedesp) [reply auto_py_to_exe/web/js/i18n.js:243]: Damn, I added an apostrophe too many. Yes, I translated it myself

File: auto_py_to_exe/web/js/i18n.js (hunk 2 above)
Comment (flaviedesp) [reply auto_py_to_exe/web/js/i18n.js:927]: ok

**Comment by brentvollebregt** [comment]
> Can you also add an entry to the translations table in the REAMDE.md file?

**Comment by flaviedesp** [comment]
ok, it's changed.

**Comment by brentvollebregt** [comment]
@flaviedesp I think you may have forgotten to push your latest changes?

**Comment by brentvollebregt** [comment]
Merging this into a develop branch so I can fix the syntax issues myself later

**Pull request merged**